tools/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileMain.scala

```diff
@@ -141,7 +141,7 @@ object ProfileMain extends Logging {
       if (appArgs.compare()) { // Compare Applications
 
         textFileWriter.write("### A. Compare Information Collected ###")
-        val compare = new CompareApplications(apps, textFileWriter)
+        val compare = new CompareApplications(apps, Some(textFileWriter))
```

Comment:
so are we printing all the health checks in different tables when doing comapre mode?  I thought we were going to put those into a single table which multiple app index, it seem inconsistent right now so I think we should make ti consistent

Reply:
Updated. It prints in a single table with multiple app index.